reflect.go

```diff
@@ -67,16 +66,42 @@ func inspect(t reflect.Type, jsonTag string) Property {
 		name := makeName(p.GoType)
 		p.Ref = makeRef(name)
 
+	case reflect.Map:
+		p.Type = "object"
+		p.AddPropertie = buildMapType(p.GoType)
+		// get the go reflect type of the map value
+		p.GoType = p.AddPropertie.GoType
+
 	case reflect.Slice:
 		p.Type = types.Array.String()
 		p.Items = &Items{}
 
-		p.GoType = t.Elem() // dereference the slice
+		// dereference the slice,get the element object of the slice
+		// Example:
+		//    t ==> *[]*Struct|Struct|*string|string
+		//    p.GoType.Kind() ==> []*Struct|Struct|*string|string
+		if p.GoType.Kind() == reflect.Slice {
```

Comment:
Is it a repeated judgment?
The outer switch has determined Slice.

Reply:
ok resolved it